primitive-types/src/lib.rs

```diff
@@ -121,6 +121,19 @@ impl From<U256> for U512 {
 	}
 }
 
+impl From<U256> for U128 {
```

Comment:
I'm not a fan of panicking `From` conversions, it doesn't make any sense for me to have them in a general-purpose library like this.

Reply:
Sorry replying late on this. The issue is that we currently already have `panic!` in a conversion that is actually needed `From<U512> for U256` in `evm`. Maybe we can keep this for now, and change everything to `TryFrom` once that's stabilized?

Reply:
Why not have our own `TryFrom` trait specfic for this crate and migrate to the std one when it's stabilises?
What I mean is that here we introduce new code that we know is wrong and we say that we are going to fix it in the unspecficied future. I'm just a little bit afraid that it might bite us in the ass instead.

Reply:
Added `TryFrom` trait and removed `panic!`